packages/web-react/src/components/Dropdown/demo/DropdownDefault.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Ref } from 'react';
+import { DropdownRenderProps } from '../../../types';
+import { Button } from '../../Button';
+import { Icon } from '../../Icon';
+import { Text } from '../../Text';
+import { Dropdown } from '..';
+
+const DropdownDefault = () => {
+  const dropdownTrigger = ({ trigger: { className, ref, ...restOf } }: DropdownRenderProps) => (
+    <Button UNSAFE_className={className} ref={ref as Ref<HTMLButtonElement>} {...restOf}>
+      Button as anchor
+    </Button>
+  );
+
+  const dropdownContent = (
```

Comment:
What about some `DropdownContentFactory` component? I think this content is repeated, isn't it?

Reply:
good point, created new `<DropdownContentFactory />` component